README.md

```diff
@@ -12,15 +33,21 @@ A minimalist library for Roman numeral operations.
 - Get Roman numerals within a range 📡
 
 ## 📦 Installation
-
-It can be installed with npm.
+To get started with ToRoman,
+ you need NPM already installed or you can do
+```sh
+  npm install
+  ```
+Next, type :
 
 ```sh
 npm i toroman
 ```
 
 ## 📥 Usage
 
+Using the ToRoman package is very simple and elaborate.
```

Comment:
Aren't simple and elaborate opposites?

Reply:
nope they aren't 
elaborate is well detailed
simple is easy and straightforward